install/installer/scripts/kots-install.sh

```diff
@@ -0,0 +1,345 @@
+#!/bin/sh
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+# shellcheck disable=SC2050,SC2153
+
+set -e
```

Comment:
Did a line by line run through, and it matches with the previous `gitpod-installer-job.yaml`